<!DOCTYPE html>
<!-- saved from url=(0014)about:internet -->
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>Systems Biology Analysis of 3T3-L1 Differentiation</title>

<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 12px;
   margin: 8px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 { 
   font-size:2.2em; 
}

h2 { 
   font-size:1.8em; 
}

h3 { 
   font-size:1.4em; 
}

h4 { 
   font-size:1.0em; 
}

h5 { 
   font-size:0.9em; 
}

h6 { 
   font-size:0.8em; 
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre {	
   margin-top: 0;
   max-width: 95%;
   border: 1px solid #ccc;
   white-space: pre-wrap;
}

pre code {
   display: block; padding: 0.5em;
}

code.r, code.cpp {
   background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * { 
      background: transparent !important; 
      color: black !important; 
      filter:none !important; 
      -ms-filter: none !important; 
   }

   body { 
      font-size:12pt; 
      max-width:100%; 
   }
       
   a, a:visited { 
      text-decoration: underline; 
   }

   hr { 
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote { 
      padding-right: 1em; 
      page-break-inside: avoid; 
   }

   tr, img { 
      page-break-inside: avoid; 
   }

   img { 
      max-width: 100% !important; 
   }

   @page :left { 
      margin: 15mm 20mm 15mm 10mm; 
   }
     
   @page :right { 
      margin: 15mm 10mm 15mm 20mm; 
   }

   p, h2, h3 { 
      orphans: 3; widows: 3; 
   }

   h2, h3 { 
      page-break-after: avoid; 
   }
}

</style>





</head>

<body>
<h1>Systems Biology Analysis of 3T3-L1 Differentiation</h1>

<p>This was based on the tools learned about in the Coursera course.  This code was most recently run on Sun Oct 27 15:50:21 2013 and is stored at /Users/davebridges/Documents/Research/differentiation.  </p>

<h2>Dataset Acquisition</h2>

<pre><code>## Parsing....
</code></pre>

<p>This file was processed using R (<a href="http://www.R-project.org/"></a>) and knitr (<a href="http://yihui.name/knitr/">Andrew et al. 2013</a>; <a href="http://yihui.name/knitr/">Xie, 2013</a>; <a href="http://www.crcpress.com/product/isbn/9781466561595">Xie, 2013b</a>).</p>

<p>The data was obtained from GEO using the dataset in GEO accession GSE6794, which is described in (<a href="http://dx.doi.org/10.1016/j.cmet.2007.01.005">Cheung et al. 2007</a>).  This file is described as:</p>

<p><strong>3T3-L1 fibroblasts are a commonly used in vitro model for adipogenesis.  When induced with hormones, they differentiate into mature fat cells.  Here, microarrays were used to study 3T3-L1 adipose differentiation through time.</strong></p>

<p>The file was downloaded, annotated and reformatted into an input file for GATE (<a href="http://dx.doi.org/10.1093/bioinformatics/btp628">MacArthur et al. 2009</a>).  This file was saved as <strong>Differentiation Time Course.clu</strong>.  The file was downloades using bioMart (<a href="">Durinck et al. 2009</a>; <a href="">Durinck et al. 2005</a>).  The annotation was done using GEOquery (<a href="">Davis &amp; Meltzer, 2007</a>).</p>

<h2>References</h2>

<ul>
<li>Steffen Durinck, Paul Spellman, Ewan Birney, Wolfgang Huber,   (2009) Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt.  <em>Nature Protocols</em>  <strong>4</strong>  1184-1191</li>
<li>Steffen Durinck, Yves Moreau, Arek Kasprzyk, Sean Davis, Bart  De Moor, Alvis Brazma, Wolfgang Huber,   (2005) BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis.  <em>Bioinformatics</em>  <strong>21</strong>  3439-3440</li>
<li>Kevin J. Cheung, Iphigenia Tzameli, Pavlos Pissios, Ilsa Rovira, Oksana Gavrilova, Toshio Ohtsubo, Zhu Chen, Toren Finkel, Jeffrey S. Flier, Jeffrey M. Friedman,   (2007) Xanthine Oxidoreductase is A Regulator of Adipogenesis And PparÎ³ Activity.  <em>Cell Metabolism</em>  <strong>5</strong>  115-128  <a href="http://dx.doi.org/10.1016/j.cmet.2007.01.005">10.1016/j.cmet.2007.01.005</a></li>
<li>Sean Davis, Paul Meltzer,   (2007) GEOquery: a bridge between the Gene Expression Omnibus (GEO) and BioConductor.  <em>Bioinformatics</em>  <strong>14</strong>  1846-1847</li>
<li>Alastair Andrew, Alex Zvoleff, Brian Diggs, Cassio Pereira, Hadley Wickham, Heewon Jeon, Jeff Arnold, Jeremy Stephens, Jim Hester, Joe Cheng, Jonathan Keane, J.J. Allaire, Johan Toloe, Kohske Takahashi, Michel Kuhlmann, Nacho Caballero, Nick Salkowski, Noam Ross, Ramnath Vaidyanathan, Richard Cotton, Romain Francois, Sietse Brouwer, Simon Bernard, Taiyun Wei, Thibaut Lamadon, Tom Torsney-Weir, Trevor Davis, Weicheng Zhu, Wush Wu, Yihui Xie,   (2013) knitr: A general-purpose package for dynamic report generation in R.  <a href="http://yihui.name/knitr/"><a href="http://yihui.name/knitr/">http://yihui.name/knitr/</a></a></li>
<li>Yihui Xie,   (2013) Dynamic Documents with {R} and knitr.  <a href="http://yihui.name/knitr/"><a href="http://yihui.name/knitr/">http://yihui.name/knitr/</a></a></li>
<li>Yihui Xie,   (2013) knitr: A Comprehensive Tool for Reproducible Research in {R}.  <a href="http://www.crcpress.com/product/isbn/9781466561595"><a href="http://www.crcpress.com/product/isbn/9781466561595">http://www.crcpress.com/product/isbn/9781466561595</a></a></li>
<li>B. D. MacArthur, A. Lachmann, I. R. Lemischka, A. Ma&#39;ayan,   (2009) Gate: Software For The Analysis And Visualization of High-Dimensional Time Series Expression Data.  <em>Bioinformatics</em>  <strong>26</strong>  143-144  <a href="http://dx.doi.org/10.1093/bioinformatics/btp628">10.1093/bioinformatics/btp628</a></li>
<li>R Core Team ,   (2013) R: A Language and Environment for Statistical Computing.  <a href="http://www.R-project.org/"><a href="http://www.R-project.org/">http://www.R-project.org/</a></a></li>
</ul>

<h2>Session Information</h2>

<pre><code>## R version 3.0.2 (2013-09-25)
## Platform: x86_64-apple-darwin10.8.0 (64-bit)
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] parallel  stats     graphics  grDevices utils     datasets  methods  
## [8] base     
## 
## other attached packages:
## [1] biomaRt_2.18.0      knitcitations_0.5-0 bibtex_0.3-6       
## [4] GEOquery_2.28.0     Biobase_2.22.0      BiocGenerics_0.8.0 
## [7] knitr_1.5          
## 
## loaded via a namespace (and not attached):
## [1] digest_0.6.3   evaluate_0.5.1 formatR_0.9    httr_0.2      
## [5] RCurl_1.95-4.1 stringr_0.6.2  tools_3.0.2    XML_3.95-0.2  
## [9] xtable_1.7-1
</code></pre>

</body>

</html>

