Systems Biology Analysis of 3T3-L1 Differentiation
========================================================

This was based on the tools learned about in the Coursera course.  This code was most recently run on `r date()` and is stored at `r getwd()`.  

Dataset Acquisition
--------------------

```{r data-entry, echo=FALSE, message=FALSE, warning=FALSE}
library(GEOquery)
library(knitcitations)
biblio <- read.bib("citations.bib")
geo_accession <- 'GSE6794'
gse <- getGEO(geo_accession, destdir=getwd(),GSEMatrix=FALSE)

#get the probesets
probesets <- Table(GPLList(gse)[[1]])$ID

data.matrix <- do.call(cbind,lapply(GSMList(gse),function(x)
 {tab <- Table(x)
 mymatch <- match(probesets,tab$ID_REF)
 return(tab$VALUE[mymatch])
 }))

data.matrix <- apply(data.matrix,2,function(x) {as.numeric(as.character(x))})
data.matrix <- log2(data.matrix)

require(Biobase)
rownames(data.matrix) <- probesets
colnames(data.matrix) <- names(GSMList(gse))
pdata <- data.frame(samples=names(GSMList(gse)))
rownames(pdata) <- names(GSMList(gse))
pheno <- as(pdata,"AnnotatedDataFrame")
eset2 <- new('ExpressionSet',exprs=data.matrix,phenoData=pheno)

require(biomaRt)
ensembl.mm = useMart('ensembl',dataset="mmusculus_gene_ensembl")
gene_list <- getBM(attributes=c('affy_mu11ksuba', 'wikigene_name'), filters = 'affy_mu11ksuba', values = probesets, mart = ensembl.mm)
annotated.data.matrix <- merge(gene_list, as.data.frame(data.matrix), by.x='affy_mu11ksuba', by.y=0)

#reassemble into a new dataframe
combined.data <- data.frame(
  Gene.Name = annotated.data.matrix$wikigene_name,
  Preconfluent = rowMeans(annotated.data.matrix[,c('GSM156862','GSM156872')], na.rm=T),
  Confluent = rowMeans(annotated.data.matrix[,c('GSM156863','GSM156873')], na.rm=T),
  Diff.6h = rowMeans(annotated.data.matrix[,c('GSM156864','GSM156874')], na.rm=T),
  Diff.12h = rowMeans(annotated.data.matrix[,c('GSM156865','GSM156875')], na.rm=T),
  Diff.24h = rowMeans(annotated.data.matrix[,c('GSM156866','GSM156876')], na.rm=T),
  Diff.2d = rowMeans(annotated.data.matrix[,c('GSM156867','GSM156877')], na.rm=T), 
  Diff.3d = rowMeans(annotated.data.matrix[,c('GSM156868','GSM156878')], na.rm=T),
  Diff.4d = rowMeans(annotated.data.matrix[,c('GSM156869','GSM156879')], na.rm=T),
  Diff.7d = rowMeans(annotated.data.matrix[,c('GSM156870','GSM156880')], na.rm=T),
  Diff.28d = rowMeans(annotated.data.matrix[,c('GSM156871','GSM156881')], na.rm=T)   )


#remove duplicates
unique.combined.data <- combined.data[!(duplicated(combined.data$Gene.Name)),]
#remove incomplete cases
unique.complete.data <- unique.combined.data[complete.cases(unique.combined.data),]
#save as clu file
gate_input_filename <- 'Differentiation Time Course.clu'
write.table(unique.complete.data, gate_input_filename, row.names=FALSE,col.names=FALSE, sep=",", quote=F)
```
This file was processed using R `r citep(biblio['stats'])` and knitr `r citep(biblio[c('knitr1','knitr2','knitr3')])`.


The data was obtained from GEO using the dataset in GEO accession `r geo_accession`, which is described in `r citep("10.1016/j.cmet.2007.01.005")`.  This file is described as:

**`r Meta(gse)$summary[1]`**

The file was downloaded, annotated and reformatted into an input file for GATE `r citep("10.1093/bioinformatics/btp628")`.  This file was saved as **`r gate_input_filename`**.  The file was downloades using bioMart `r citep(biblio[c('biomaRt1', 'biomaRt2')])`.  The annotation was done using GEOquery `r citep(biblio['GEOquery'])`.

Gene Expression Changes With Differentiation
---------------------------------------------

```{r differential-expression, warning=FALSE, echo=FALSE, message=FALSE}
require(limma)
pData(eset2)$treaments <- rep(colnames(combined.data)[2:11],2)
treatments <- rep(colnames(combined.data)[2:11],2)
design <- model.matrix(~0+treatments, eset2)

#fit to that design matrix
fit <- lmFit(eset2, design)
cont.diff <- makeContrasts("treatmentsDiff.7d-treatmentsConfluent",levels=design)
fit.diff <- contrasts.fit(fit, cont.diff)
fit.diff <- eBayes(fit.diff)

cont.conf <- makeContrasts("treatmentsConfluent-treatmentsPreconfluent",levels=design)
fit.conf <- contrasts.fit(fit, cont.conf)
fit.conf <- eBayes(fit.conf)
```

Time course dependent gene expression changes were done using limma `r citep(biblio['limma'])`.


### Effects of Differentiation
```{r table-differentiation, results='asis', echo=FALSE, message=FALSE}
require(xtable)
print(xtable(merge(topTable(fit.diff, adjust="BH", sort.by="p"), gene_list, by.x=0, by.y='affy_mu11ksuba')), type="html")
```

### Effects of Cellular Confluence
```{r table-confluence, results='asis', echo=FALSE}
print(xtable(merge(topTable(fit.conf, adjust="BH", sort.by="p"), gene_list, by.x=0, by.y='affy_mu11ksuba')), type="html")
```

### Some Sample Genes
```{r barplot, echo=FALSE}
gene.list <- c("Adipoq","Inpp4a","Tbc1d1","Med20","Cdipt")
for (gene in gene.list) {
barplot(as.numeric(unique.complete.data[unique.complete.data$Gene.Name==gene,2:10]),
        main = gene,
        las=2,
        names.arg=colnames(unique.complete.data)[2:10])
}
```

Short Term Differentiation Dataset
-----------------------------------

```{r short-term, echo=FALSE, message=FALSE, warning=FALSE}
# load series and platform data from GEO

gset <- getGEO("GSE29744", GSEMatrix =TRUE, destdir=getwd())
if (length(gset) > 1) idx <- grep("GPL7202", attr(gset, "names")) else idx <- 1
gset <- gset[[idx]]

# make proper column names to match toptable 
fvarLabels(gset) <- make.names(fvarLabels(gset))

# group names for all samples
sml <- c("G0","G0","G0","X","X","X","G1","G1","G1","X","X","X");

# eliminate samples marked as "X"
sel <- which(sml != "X")
sml <- sml[sel]
gset <- gset[ ,sel]

# log2 transform
ex <- exprs(gset)
qx <- as.numeric(quantile(ex, c(0., 0.25, 0.5, 0.75, 0.99, 1.0), na.rm=T))
LogC <- (qx[5] > 100) ||
          (qx[6]-qx[1] > 50 && qx[2] > 0) ||
          (qx[2] > 0 && qx[2] < 1 && qx[4] > 1 && qx[4] < 2)
if (LogC) { ex[which(ex <= 0)] <- NaN
  exprs(gset) <- log2(ex) }

# set up the data and proceed with analysis
fl <- as.factor(sml)
gset$description <- fl
design <- model.matrix(~ description + 0, gset)
colnames(design) <- levels(fl)
fit <- lmFit(gset, design)
cont.matrix <- makeContrasts(G1-G0, levels=design)
fit2 <- contrasts.fit(fit, cont.matrix)
fit2 <- eBayes(fit2)
tT <- topTable(fit2, adjust="BH")

# load NCBI platform annotation
gpl <- annotation(gset)
platf <- getGEO(gpl, AnnotGPL=TRUE, destdir=getwd())
ncbifd <- data.frame(attr(dataTable(platf), "table"))

# replace original platform annotation
tT <- tT[setdiff(colnames(tT), setdiff(fvarLabels(gset), "ID"))]
tT <- merge(tT, ncbifd, by="ID")
tT <- tT[order(tT$P.Value), ]  # restore correct order

tT <- subset(tT, select=c("ID","adj.P.Val","P.Value","t","B","logFC","Gene.symbol","Gene.title"))
write.fit(fit2, file="GSE29744 Analysis.csv", sep=",")
################################################################
#   Boxplot for selected GEO samples
library(Biobase)
library(GEOquery)

# load series and platform data from GEO

gset <- getGEO("GSE29744", GSEMatrix =TRUE, destdir=getwd())
if (length(gset) > 1) idx <- grep("GPL7202", attr(gset, "names")) else idx <- 1
gset <- gset[[idx]]

# group names for all samples in a series
sml <- c("G0","G0","G0","X","X","X","G1","G1","G1","X","X","X")

# eliminate samples marked as "X"
sel <- which(sml != "X")
sml <- sml[sel]
gset <- gset[ ,sel]

# order samples by group
ex <- exprs(gset)[ , order(sml)]
sml <- sml[order(sml)]
fl <- as.factor(sml)
labels <- c("Day0","Hour6")

#extract and save significantly different genes
sig.genes <- topTable(fit2, adjust.method="BH", p.value=0.001, number=1000000, sort.by="logFC")$GENE_SYMBOL
unique.sig.genes <- droplevels(unique(sig.genes))
unique_sig_genes_filename <- "GSE29744 Significant Genes.txt"
write(as.character(unique.sig.genes),  unique_sig_genes_filename)
```

These data were published in `r citet("10.1186/1471-2199-13-9")`.  We analysed the non-polysome bound mRNA fraction comparing undifferentiated to 3T3-L1 cells treated with DMI for 6h.  Output the significantly different genes into a text file named `r unique_sig_genes_filename`.

```{r table-differentiation-short-term, results='asis', echo=FALSE, message=FALSE}
print(xtable(tT), type="html")
```


References
-----------

```{r bibliography, echo=FALSE, results='asis', message=FALSE}
library(bibtex)
write.bib(c('biomaRt', 'GEOquery', 'knitcitations', 'Biobase', 'stats', 'knitr','limma'), file="citations.bib")
bibliography("html")
```

Session Information
---------------------


```{r session-info, echo=FALSE}
sessionInfo()
```
